package.json

```diff
@@ -49,6 +51,8 @@
     "typecheck": "pnpm turbo typecheck --no-daemon",
     "desktop": "pnpm --filter ledger-live-desktop",
     "cli": "pnpm --filter live-cli",
+    "coin": "pnpm --filter coin-framework",
+    "coin:polkadot": "pnpm --filter coin-polkadot",
```

Comment:
Do you use this one? (honest question)
If so we can keep it, otherwise it's nice to have but I predict it won't scale as no integrator will think of adding theirs here, so it could be removed, just for the sake of keeping this list clean.